tests/tests/encoder.rs

```diff
@@ -8,6 +8,14 @@ static DROP_ENCODER: GpuTestConfiguration = GpuTestConfiguration::new().run_sync
     drop(encoder);
 });
 
+#[gpu_test]
+static DROP_QUEUE_BEFORE_CREATING_COMMAND_ENCODER: GpuTestConfiguration =
+    GpuTestConfiguration::new().run_sync(|ctx| {
```

Comment:
You can mark a test as always failing, see https://wgpu.rs/doc/wgpu_test/struct.GpuTestConfiguration.html#method.parameters and https://wgpu.rs/doc/wgpu_test/struct.TestParameters.html#method.expect_fail